### PR TITLE
Allow patterns to be passed to rails benchmarks

### PR DESCRIPTION
**Pull request by klgilbert**

### CODE REVIEW

**Comment by tgxworld** [comment]
@klgilbert Tests are failing :smiley_cat:

File: app/jobs/remote_server_job.rb [hunk 1]
```diff
@@ -96,6 +96,8 @@ def ruby_trunk_discourse(ssh, commit_hash, options)
   end
 
   def rails_releases(ssh, rails_version, options)
+    include_patterns = build_include_patterns(options[:include_patterns])
```
Comment (tgxworld) [inline app/jobs/remote_server_job.rb]: I just realized we don't need to assign it to a variable. Could we update this for the other jobs too? Thanks!

Comment (klgilbert) [inline app/jobs/remote_server_job.rb]: Do you want me to remove the build_include_patterns method and just call -e \"INCLUDE_PATTERNS=... directly

Comment (tgxworld) [inline app/jobs/remote_server_job.rb]: The method is called in a few other methods but there isn't a need to assign it to a variable.

Date: Wed, 17 Jun 2015 09:18:33 -0700

**Comment by tgxworld** [comment]
Thanks!

**Pull request merged**